pyproject.toml

```diff
@@ -19,17 +19,19 @@ notebook = "^6.5.4"
 pandas = "^2.0.1"
 nltk = "^3.8.1"
 scikit-learn = "^1.2.2"
-dslinter = "^2.0.9"
```

Comment:
Was dslinter causing dependency issues?

Reply:
No, it didn't cause any dependency issues, but since `model-training` doesn't import it anywhere. It's more appropriate to put it in the dev-dependencies.